pennylane/grouping/transformations.py

```diff
@@ -107,15 +107,15 @@ def diagonalize_pauli_word(pauli_word):
     return diag_term
 
 
-def diagonalize_qwc_grouping(qwc_grouping):
+def diagonalize_qwc_pauli_words(qwc_grouping):
```

Comment:
:+1: This definitely makes more sense after reading the first line of the docstring.